backend/frontend/views.py

```diff
@@ -294,6 +294,24 @@ def action_start_card(request, card_id):
         },
     )
 
+@user_passes_test(is_super)
+@csrf_exempt
+def link_card_details_page(request, card_id):
+    """The card is in the backlog and the user has chosen to start it"""
+    card = get_object_or_404(AgileCard, id=card_id)
+
+    recruit_project_reviews = get_list_or_404(RecruitProjectReview)
```

Comment:
only get the reviews that are associated with the current project. Getting ALL the reviews is not going to work out.

```
project.project_reviews.order_by("- timestamp")
```

And I believe that get_list_or_404 will only raise a 404 if there are literally zero reviews in the database. This is a misuse of get_list_or_404